src/common.rs

```diff
@@ -3,6 +3,7 @@
 use crate::{NetworkData, Networks, NetworksExt, UserExt};
 
 /// Trait to have a common fallback for the [`Pid`][crate::Pid] type.
+#[deprecated]
```

Comment:
Please don't add this.

Reply:
For more context: this trait is very convenient in case a `Pid` isn't an integer (like it should be for unsupported targets, I'll send a PR after this one is merged to fix it).